src/App.js

```diff
@@ -1,25 +1,93 @@
-import logo from './logo.svg';
+import axios from 'axios';
 import './App.css';
+import { useState } from 'react';
+import Header from './Header.js';
+import Footer from './Footer.js'
+import Input from './Input.js'
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+
+// store API and URL info
+
+const apiKey = '11835b08a653433e998a492061031a38';
+const baseUrl= 'https://api.weatherbit.io/v2.0/current';
+
+// useStates
+
+const [weather, setWeather] = useState([]);
+const [userInput, setUserInput] = useState('');
+
+// call API to BitWeather & throw errors if needed
+const fetchWeatherData = async(city) => {
+    axios({
+      method: 'GET',
+      url: baseUrl,
+      dataResponse: 'json',
+      params: {
+        key: apiKey,
+        country: 'Canada',
+        city: city
+      }
+    }).then( (response) => {
+      if(response.request.statusText === 'OK'){
+        setWeather(response.data.data);
+      } else {
+        throw new Error(response.request.statusText)
+      }
+    }).catch((error) => { 
+      if (error.message !== 'OK')
+      alert('We are sorry your city was not found')
```

Comment:
an actual message in app could be another solution to letting users know to try again.
I read that using alert in general is not commonplace any more. again, I could be wrong though hahaha